third_party/2/requests/models.pyi

```diff
@@ -77,10 +77,10 @@ class Request(RequestHooksMixin):
     def prepare(self): ...
 
 class PreparedRequest(RequestEncodingMixin, RequestHooksMixin):
-    method = ...  # type: Any
-    url = ...  # type: Any
-    headers = ...  # type: Any
-    body = ...  # type: Any
+    method = ...  # type: str
```

Comment:
All of those are technically `Optional[]` since they are only initialized to `None` in the constructor.